tests/test_concept_graph.py

```diff
@@ -200,11 +200,11 @@ def test_to_from_dict():
     node_dict['somekey'] = 'foo'
     with pytest.raises(AttributeError) as e:
         ConceptNode.from_dict(node_dict)
-        assert 'concept_attribute_pair' in str(e)
+    assert 'somekey' in str(e.value)
```

Comment:
Why not what it was before, `concept_attribute_pair`?

Reply:
because it's wrong, lol

Reply:
ohhh woops i didn't see line 200. I was like where did `somekey` come from lol